### PR TITLE
Bugs are fixed, dockerfiles are added and backend for portfolio is al…

### PR DESCRIPTION
**Pull request by lakshyabatman**
…so added

### CODE REVIEW

File: server/controller/profile.controller.js [hunk 1]
```diff
@@ -244,7 +235,9 @@ module.exports = {
       .then((result)=>{
         if(result.isEmpty() === true){
           result.array().forEach((error)=>{
-            res.status(400).send(error)
+            res.status(400).json({
```
Comment (vaibhavdaren) [inline server/controller/profile.controller.js]: sad

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+![enter image description here](https://www.opportunitiesforafricans.com/wp-content/uploads/2018/03/google-summer-of-code-2018.png)
+
+# Development of modules with new UI/UX
```
Comment (vaibhavdaren) [inline README.md]: write this in blog :D 
nice work

File: README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+# Development of modules with new UI/UX
+
+### This branch contains all the work by Lakshya Khera during Google Summer Of Code 2019.
+
+[Link to project on GSoC website](https://summerofcode.withgoogle.com/projects/#6432124159131648)
+<br>
+[Blog Posts](https://lakshyakhera.wordpress.com/category/gsoc/)
+<br>
+[Commits authored by me under GSOC ](commits.txt)
```
Comment (vaibhavdaren) [inline README.md]: remove this file

File: commits.txt [hunk 4]
```diff
@@ -0,0 +1,244 @@
+commit 63a0b8f91168e3c5cefbf74b36539ebc5b12e9dd
+Author: lakshyabatman <lakshya.khera@gmail.com>
+Date:   Thu Aug 22 19:16:26 2019 +0530
```
Comment (vaibhavdaren) [inline commits.txt]: remove this

Comment (lakshyabatman) [reply commits.txt]: done

File: server/controller/authcontroller.js [hunk 5]
```diff
@@ -5,12 +5,12 @@ const Social=require('../schema/social.js')
 const validateRegisterInput=require('../validation/registervalidation.js')
 const validateLoginInput=require('../validation/loginvalidation.js')
 const jwt=require('jsonwebtoken')
-const bcrypt = require('bcrypt')
+const bcrypt = require('bcryptjs')
 const secret ='mySecret'
 const _ =require('lodash')
 let clientBaseURL = 'http://localhost:8080/feed/'
```
Comment (vaibhavdaren) [inline server/controller/authcontroller.js]: needs change

**Pull request merged**